report/renderer/components.js

```diff
@@ -4,27 +4,27 @@
 
 /** @typedef {import('./dom.js').DOM} DOM */
 
-/* eslint-disable max-len */
+/* eslint-disable max-len, quotes, comma-spacing */
```

Comment:
LGTM

but maybe we should just disable eslint entirely?

Reply:
I was originally going to do that but then I saw how few rules were needed. It's nice to have it close to our lint style since it's little effort to get it there. If it ever really needs to diverge it's no big deal to disable, of course.